endtoend_tests/conftest.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2020 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Forseti end-to-end test configuration"""
+
+import pytest
+import time
+from sqlalchemy import create_engine
+
+CLOUDSQL_PORT = 3306
+FORSETI_SERVER_CONFIG_PATH = (
+    '/home/ubuntu/forseti-security/configs/forseti_conf_server.yaml')
+
+
+def pytest_addoption(parser):
+    parser.addoption('--cai_dump_file_gcs_paths',
+                     default='',
+                     help='CAI dump file paths for inventory performance test')
+    parser.addoption('--cloudsql_password', help='Cloud SQL password')
+    parser.addoption('--cloudsql_port',
+                     default=CLOUDSQL_PORT,
+                     help='Cloud SQL port')
+    parser.addoption('--cloudsql_username', help='Cloud SQL username')
+    parser.addoption('--forseti_server_bucket_name',
+                     help='Forseti server bucket name')
+    parser.addoption('--forseti_server_config_path',
+                     default=FORSETI_SERVER_CONFIG_PATH,
+                     help='Path to Forseti server config')
+    parser.addoption('--root_resource_id',
+                     help='Root resource id for inventory performance test')
+
+
+def pytest_configure(config):
+    config.addinivalue_line(
+        'markers', 'e2e: mark test to run only on named environment'
+    )
+    config.addinivalue_line(
+        'markers', 'inventory: mark test to run only on named environment'
+    )
+
+
+@pytest.fixture
```

Comment:
General pytest configuration will add some options and provide those as fixtures for the tests. There is also a CloudSQL connection made to the Forseti DB to run queries for test assertions.